packages/types/src/extension-types.ts

```diff
@@ -75,24 +75,37 @@ export interface IEvent {
   from?: Identity.IIdentity;
 }
 
-/** Identification of extensions handled by this implementation */
-export enum ID {
+/**
+ * Identification of extensions handled by this implementation.
+ * */
+export enum OTHER_ID {
   CONTENT_DATA = 'content-data',
-  PAYMENT_NETWORK_BITCOIN_ADDRESS_BASED = 'pn-bitcoin-address-based',
-  PAYMENT_NETWORK_TESTNET_BITCOIN_ADDRESS_BASED = 'pn-testnet-bitcoin-address-based',
-  PAYMENT_NETWORK_ERC20_ADDRESS_BASED = 'pn-erc20-address-based',
-  PAYMENT_NETWORK_ERC20_PROXY_CONTRACT = 'pn-erc20-proxy-contract',
-  PAYMENT_NETWORK_ERC20_FEE_PROXY_CONTRACT = 'pn-erc20-fee-proxy-contract',
-  PAYMENT_NETWORK_ERC777_STREAM = 'pn-erc777-stream',
-  PAYMENT_NETWORK_ETH_FEE_PROXY_CONTRACT = 'pn-eth-fee-proxy-contract',
-  PAYMENT_NETWORK_ETH_INPUT_DATA = 'pn-eth-input-data',
-  PAYMENT_NETWORK_NATIVE_TOKEN = 'pn-native-token',
-  PAYMENT_NETWORK_ANY_TO_NATIVE_TOKEN = 'pn-any-to-native-token',
-  PAYMENT_NETWORK_ANY_DECLARATIVE = 'pn-any-declarative',
-  PAYMENT_NETWORK_ANY_TO_ERC20_PROXY = 'pn-any-to-erc20-proxy',
-  PAYMENT_NETWORK_ANY_TO_ETH_PROXY = 'pn-any-to-eth-proxy',
 }
 
+export enum PAYMENT_NETWORK_ID {
```

Comment:
Strong impact required for the main change: instead of mapping a new enum out of `ID` to qualify payment network IDs, we now have 2 enums.